src/converters/byteConverter.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Byte Converter Utility
+ * Converts between Bytes, KB, MB, GB, and TB
+ *
+ * Example:
+ *   convertBytes(1024, "KB", "MB") => 1
+ */
+
+export type ByteUnit = "B" | "KB" | "MB" | "GB" | "TB";
+
+export function convertBytes(value: number, fromUnit: ByteUnit, toUnit: ByteUnit): number {
```

Comment:
Are you expecting `value` to be a negative number? . If not, you can add a validation for that.